src/plugins/share/public/components/tabs/export/export_content.tsx

```diff
@@ -218,19 +237,35 @@ const ExportContentUi = ({
     );
   };
 
+  const renderWarnings = (warning: { title: string; message: string }) => (
+    <>
+      <EuiSpacer size="m" />
+      <EuiCallOut color="warning" iconType="warning" title={warning.title}>
+        {warning.message}
+      </EuiCallOut>
+    </>
+  );
+
   return (
     <>
       <EuiForm>
-        <>{helpText}</>
-        <EuiSpacer size="m" />
-        <>{renderRadioOptions()}</>
-        {renderHelpText()}
-        <EuiSpacer size="xl" />
+        <EuiText size="s">
+          {helpText ?? (
+            <FormattedMessage
+              id="share.export.helpText"
+              defaultMessage="Select the file type you would like to export for this visualization."
+            />
+          )}
+        </EuiText>
```

Comment:
Add `EuiText` wrapper to match `Links` tab. Before the text was slightly different between tabs.